bundle-generation/generate-charts.py

```diff
@@ -620,7 +619,7 @@ def updateHelmResources(chartName, helmChart, exclusions, inclusions, branch):
 
                 if kind in namespace_scoped_kinds:
                     current_namespace = resource_data['metadata'].get('namespace', None)
-                    if current_namespace is None:
+                    if current_namespace is None or chartName == 'flight-control':
```

Comment:
```suggestion
                    if current_namespace is None:
```

I don't believe we need to specify if the `chartName` is equal to `flight-control`. If the `flight-control` resource has a namespace already defined in the template resource, our operator will either use that namespace or the one in `.Values.global.namespace`